doc/WebSite/Feature-Management.md

```diff
@@ -248,14 +248,14 @@ to get current values of the features.
 
 ### Feature Manager
 
-If you need to definitions of features, you can inject and use
+If you need the definitions of features, you can inject and use
 **IFeatureManager**.
 
 ### A Note For Editions
 
-ASP.NET Boilerplate framework has not a built-in edition system because
+The ASP.NET Boilerplate framework does not have a built-in edition system because
 such a system requires a database (to store editions, edition features,
-tenant-edition mappings and so on...). Therefore, edition system is
+tenant-edition mappings and so on...). Therefore, the edition system is
 implemented in [module zero](/Pages/Documents/Zero/Edition-Management).
-You can use it to easily have an edition system, or you can implement
-all yourself.
+You can use it to easily have an edition system or you can implement
```

Comment:
"You can use it ~`to easily have an`~ `as a ready-made` edition system`,` or ~`you can`~ implement one yourself."